pages/clusters/overview.md

```diff
@@ -1,18 +1,40 @@
 # Clusters overview
 
-Clusters are a new way of managing your Buildkite agents. They allow teams to self-manage their agent pools, let admins create isolated sets of agents and pipelines within the one Buildkite organization, and help to make agents and queues more discoverable across your organization.
+>📘 Clusters
+> Clusters will be enabled for all organizations on February 26 2024.
+
+Clusters is a Buildkite feature used to manage and organize agents and queues, which:
+
+- allows teams to self-manage their Buildkite agent pools,
+- allows admins to create isolated sets of agents and pipelines within the one Buildkite organization,
+- helps make agents and queues more discoverable across your organization, and
+- provides easily accessible queue metrics.
 
 The following diagram shows the architecture with cluster enabled.
 
 <%= image "clusters-architecture.png", alt: "Diagram showing existing architecture and architecture with clusters" %>
 
 Clusters encapsulate groups of agents and pipelines, enabling the following:
 
-* Clusters are viewable to your entire organization, allowing engineers to better understand the agents and queues available for their pipelines.
-* Individual users or teams can maintain their own clusters. Cluster maintainers can manage queues and agent tokens and add and remove pipelines.
-* Pipelines can be assigned to a cluster, ensuring their builds run only on the agents connected to this cluster. These pipelines can also trigger builds only on other pipelines in the same cluster.
+- Clusters are viewable to your entire organization, allowing engineers to better understand the agents and queues available for their pipelines.
```

Comment:
To reduce ambiguity, I generally like to clarify what term/type of entity - e.g. 'organization' is being referred to in a new paragraph of text, or near the top of a list, where this term might be used throughout the other list items (without its clarification).

Hence, my suggestion of adding 'Buildkite' before 'organization'...

```suggestion
- Clusters are viewable to your entire Buildkite organization, allowing engineers to better understand the agents and queues available for their pipelines.
```